hibike/travis/Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: install artifacts-install lint test
+
+install:
+	cd .. && wget https://downloads.arduino.cc/arduino-1.8.1-linux64.tar.xz
+	cd .. && sudo bash setup.sh "arduino-1.8.1-linux64.tar.xz"
+
+artifacts-install:
+	$(nop)
+
+lint:
+	$(nop)
+
+test:
+	cd .. && rm -rf ./bin
+	cd .. && for device in ./devices/*; do make DEVICE=$$device; done
```

Comment:
BTW I think you may need to do `rm -rf ./bin` for every device, because make may be a no-op unless you delete the stale binaries.